jsonschemagen/app/User.php

```diff
@@ -26,4 +26,8 @@ class User extends Authenticatable
     protected $hidden = [
         'password', 'remember_token',
     ];
+
+    public function projects(){
```

Comment:
This violated PSR-2. It's better if you follow the standards, not necessary for now but it really counts on my review:

```php
public function projects()
{
```

Instead of:

```php
public function projects(){
```